lib/sprockets/utils/gzip.rb

```diff
@@ -0,0 +1,64 @@
+require 'concurrent'
```

Comment:
Can we require this in the file that is using it? Also we could require only `future` file since we only need it.